### DISCUSSION TITLE
Make all the items to fit the old textures/ textures before 1.14

### DISCUSSION THREAD

**Discussion by Swifty07** [Roadhog360/Et-Futurum-Requiem#279] [Improvements]
I like this mod, however some of the items added to this mod doesn't fit 1.7.10. So I'm writing this so hopefully you can make the textures fit better. Also there's one thing that is from a mod not from vanilla, which are the shulker box upgrades, because I think you added them because of the upgraded shulker boxes mod

IDEAS (open to more suggestions):

1. Netherite armor should have a texture change to fit with the rest of the armor.
2. Glazed Terracota should be changed to Glazed Stained Clay because in 1.7.10 colored terracota is called stained clay.
3. The new 1.17 ores to fit with the old textures.

**STILL OPEN FOR IDEAS TO MAKE THE MOD TEXTURES FIT WITH THE OLD VANILLA TEXTURES**
**COMMENT YOUR IDEAS BELOW**

**Comment by Swifty07** [comment]
i just realized that hardened clay and stained clay has been renamed to terracotta

**Comment by Roadhog360** [comment]
The new 1.17 deepslate ores have already been adjusted to fit the old ores. Texture changes are generally modest and often avoided.

**Comment by Swifty07** [comment]
well I have played it and yes you're correct about the deepslate ores, but netherite armor is still in the new texture. Also one thing, when playing with copper from other mods it drops raw copper from this mod is that meant to be a bug?
![2023-05-27_15 17 11](https://github.com/Roadhog360/Et-Futurum-Requiem/assets/129383435/95596398-8da9-4684-b646-ac441fed0c2f)
![2023-05-27_15 17 08_2](https://github.com/Roadhog360/Et-Futurum-Requiem/assets/129383435/edb47be4-9bff-4e70-9c38-a2b9a25cda47)

**Comment by GregoriusT** [comment]
This is a Feature, an intended one at that. ;)

Use Silk Touch if you want the Ore Blocks

**Reply by Swifty07** [reply]
oh ok

**Comment by Roadhog360** [comment]
Revisiting many suggestions and I decided to elaborate on what guideline I try to uphold when choosing whether or not to change a texture.
So first of all, the pre-1.14 art style feels like it fits with a blend of many different artstyles, meaning I don't feel the need to change most textures. However there may be a few exceptions.

So, if it appears to stick out and I have a choice between changing textures already in the game or changing the texture of the backported content, the answer is to always transform the new stuff to fit in, not transform the old to fit with the new.
So under this logic, I changed nether gold ore to match the old netherrack. When lecterns are added, their base will be changed to use the old plank texture.
However netherite ingot and chestplate and copper ingot while they don't match with their set, were clearly meant to be special and differ from the other designs, so they were left unchanged. There's really no "side-by-side eyesore" effect you'd get like with how nether gold sticks out.
None of the other textures directly clash with other existing blocks or items either. They do have a different art style, but so does most mod textures, and it feels just fine. Hell even some textures in 1.7.10 and pre-1.14 textures seem to be made with slightly different art styles.

To me I find it only important to change things when there's a blatant and obvious clash that can't be ignored or can't be justified like the previous examples. Modded 1.7.10 is kind of a mixed pot of tons of artstyles anyhow.

Another reason is I just don't want to commit to having to rely on an artist to make these "matching" textures. I feel it'd only be more inconsistent if they ended up becoming unavailable and then *only one* backported block didn't get retextured or something. I'll likely also be incredibly picky about it. I'll want to be sure the designs provided by the artist are as objectively "1.7.10 styled". I'd want it to truly feel like an official old texture. So simply taking them from a resource pack that already does it, even with the author's explicit permission. I wouldn't want to bother them for new updates or tweaks to things. I'd want the artist committed to be making these textures specifically *for* Et Futurum Requiem.

And this is quite a big task so I just haven't bothered asking. I felt as if I described all this to someone I'd come across as choosy or something but I realize I probably have no harm in just throwing this out there. So before I just didn't bother and used vanilla textures except when I found reworks *absolutely 100% necessary and unavoidable*.

I won't be closing this since I'm not against the idea however and am open to discussing with you if you were willing to contribute a consistent supply of old style textures-- and you were good at making them look and feel like the official old style.

**Comment by Roadhog360** [comment]
Closing due to no proposed solution provided to the problem of textures being provided.
